dcs-packaging-tool/dcs-packaging-tool-gui/src/main/java/org/dataconservancy/packaging/gui/view/impl/CreateNewPackageViewImpl.java

```diff
@@ -65,7 +61,8 @@ public CreateNewPackageViewImpl(Labels labels) {
         super(labels); 
         this.labels = labels;
         
-        getContinueButton().setText(labels.get(LabelKey.CONTINUE_BUTTON));
+        getContinueButton().setText(labels.get(LabelKey.NEXT_BUTTON));
```

Comment:
There was none and the text hasn't changed, I just figured since the actual text says "Next", the constant should too. Happy to change it back.